mmv1/third_party/terraform/services/compute/resource_compute_security_policy.go.tmpl

```diff
@@ -805,6 +817,30 @@ func resourceComputeSecurityPolicyCreate(d *schema.ResourceData, meta interface{
 		return err
 	}
 
+	labels := tpgresource.ExpandLabels(d)
+	if labels != nil {
+		// To get labels_fingerprint, we need to read the resource first
+		err = resourceComputeSecurityPolicyRead(d, meta)
+		if err != nil {
+			log.Printf("[DEBUG] Error reading SecurityPolicy %q after creation: %s", sp, err)
+			return err
+		}
+		// Now we can set the labels
+		setLabels := &compute.GlobalSetLabelsRequest{
+			Labels:           labels,
+			LabelFingerprint: d.Get("label_fingerprint").(string),
+		}
+		op, err = client.SecurityPolicies.SetLabels(project, sp, setLabels).Do()
```

Comment:
Can the `labels` be included in the CREATE object, instead of being updated after the resource is created?